Content.Shared/Verbs/HideContextMenuComponent.cs

```diff
@@ -1,15 +0,0 @@
-﻿using Robust.Shared.GameObjects;
-
-namespace Content.Shared.Verbs
-{
-    [RegisterComponent]
-    public class HideContextMenuComponent : Component, IShowContextMenu
```

Comment:
why was this removed? its used on some stuff, like low wall entities

Reply:
I just changed this to be a tag instead. I can undo that and keep it as a component?

Reply:
tags fine i suppose